osmaxx/conversion/converters/converter_gis/bootstrap/bootstrap.py

```diff
@@ -48,7 +52,7 @@ def bootstrap(self):
 
     @property
     def geom(self):
-        return self._extent
+        return polyfile_helpers.parse_poly_string(self.area_polyfile_string)
```

Comment:
Should we use [`memoize`](https://pypi.python.org/pypi/memoize) for it or implement a simple memoization ourselves? Or just make it a regular method instead of a property, so that the weight of invoking it is better communicated.